README.md

```diff
@@ -32,7 +32,18 @@ To build for the EV3:
 
 * On your host computer (not in `brickstrap shell`), use NFS or sshfs to share
 <some-path-like-home/user/build-brickman> with your EV3.
-* On your EV3, connect the share and run `./brickman`
+* On your EV3, stop the runing service: `systemctl stop brickman`, connect the
+share and run `./brickman` or if you replaced the one in `/usr/sbin` restart it
```

Comment:
This is of course only for verifying the changed before you send a patch - changed wording to say "temporarilly replaced".